src/os/macos.rs

```diff
@@ -16,6 +16,13 @@ pub trait WindowExt {
     ///
     /// The pointer will become invalid when the `Window` is destroyed.
     fn get_nsview(&self) -> *mut c_void;
+
+    /// For windows created with the [blurred](WindowBuilder::with_blur) option,
+    /// this controls the appearance of the blur effect.
+    /// 
+    /// Marked as unsafe because depending on the version of macOS and the `BlurMaterial` variant passed,
+    /// this might cause a crash.
```

Comment:
Is there a lowest common denominator of supported enum variants? I don't think people will be comfortable using a method that can crash based on criteria that are only vaguely explained.

Reply:
The lowest common denominator appears to be `AppearanceBased`, `Light`, `Dark`, `MediumLight` and `UltraDark` ([source](https://developer.apple.com/documentation/appkit/nsvisualeffectview/material?changes=latest_major)), but those are now marked as deprecated. There are some marked as "modified", I think that means their underlying integer value has changed, which would disqualify them.  
This is all quite problematic since this swift enum is our only source of truth and it appears Apple is comfortable changing the order of constants willy nilly - they only keep the swift code consistent, but apparently not the enum's binary representation.

Honestly I don't know what to do here. This is why I was originally unsure about the `BlurMaterial` enum in the first place.  
The only alternative I see is exposing the raw `i64` and removing the enum, but that won't solve the crash / macOS-version dependency problem either.

Reply:
How impractical would it be for us to translate the values depending on the version of macOS?

Also, shouldn't it be `isize` instead of `i64`?

Reply:
I don't think that would be too much work, definitely doable.

Yeah you could be right with `isize` over `i64`. The right size is whatever the size of the swift enum is. `isize` should be a good bet.